README.md

```diff
@@ -29,4 +29,58 @@
 ## Documentation
 
 To learn more about our ecosystem, vision, and product specifics - visit our 
-[mdbook](https://docs.composable.finance).
\ No newline at end of file
+[mdbook](https://docs.composable.finance).
+
+
+## Nix
+
+We use [`nix`](https://nixos.org/) in order to reproducibly build our products. We recommend either installing `nix` or switching to `NixOS`. Alternatively, you can run our packages with just `docker` installed.
+Our packages support both **x86** and **ARM** architectures.
+
+Once you have `nix` or `NixOS` installed, you should enable the following features:
+```nix
+{
+  nix = {
+    useSandbox = "relaxed";
+    extraOptions = ''
+      experimental-features = nix-command flakes
+    '';
+  };
+}
```

Comment:
For non-nixos, probably easier to set `~/.config/nix/nix.conf` with 
```
experimental-features = nix-command flakes
sandbox = relaxed
```

Reply:
Will add 👍🏻

Reply:
added